### ISSUE TITLE
Message not found for provided ID on incoming push

### ISSUE THREAD

**Issue by pflenker** [urbanairship/react-native-module#200]
# Preliminary Info

### What Airship dependencies are you using?
react-native-module 3.1.1
# Report
### What unexpected behavior are you seeing?
On iOS (Possibly on Android, too), trying to react to an incoming push results in an error: `Message not found for provided id` . Calling `UrbanAirship.refreshInbox();` before opening the message fixes it.

### What is the expected behavior?
Either the documentation states that the inbox needs to be refreshed before an incoming push can be opened, or the error does not occur and the message opens.

### What are the steps to reproduce the unexpected behavior?
- Call `UrbanAirship. setAutoLaunchDefaultMessageCenter(false)` to start receiving the appropriate events
- Subscribe to the event as follows:
```
 UrbanAirship.addListener('showInbox', (push) => {
    if (push.messageId) {
      UrbanAirship.displayMessage(push.messageId, true);
    }
});
```
- Run this on iOS. Background the app, send a push notification and tap on it immediately. Observe the error.
### Do you have logging for the issue?
```
Possible Unhandled Promise Rejection (id: 1):
Error: Message not found for provided id.
Error: Message not found for provided id.
    at createErrorFromErrorData (blob:http://localhost:8081/73ddccf9-146b-449b-b025-95d37fcdbfb9:2118:17)
    at blob:http://localhost:8081/73ddccf9-146b-449b-b025-95d37fcdbfb9:2070:27
    at MessageQueue.__invokeCallback (blob:http://localhost:8081/73ddccf9-146b-449b-b025-95d37fcdbfb9:2637:18)
    at blob:http://localhost:8081/73ddccf9-146b-449b-b025-95d37fcdbfb9:2368:18
    at MessageQueue.__guard (blob:http://localhost:8081/73ddccf9-146b-449b-b025-95d37fcdbfb9:2541:13)
    at MessageQueue.invokeCallbackAndReturnFlushedQueue (blob:http://localhost:8081/73ddccf9-146b-449b-b025-95d37fcdbfb9:2367:14)
    at http://localhost:8081/debugger-ui/debuggerWorker.js:80:58
```

**Comment by rlepinski** [comment]
This is actually an issue with Android as well. Luckily we have made changes to the underlying SDK where we no longer need the message instance to be fetched before trying to display it. Will fix in next release.

Are you able to update to 0.6 with AndroidX?

**Comment by pflenker** [comment]
> Are you able to update to 0.6 with AndroidX?

Unfortunately not for the next 3 months at least..

**Comment by rlepinski** [comment]
Fixed in 3.2.0
